hudi-client/hudi-spark-client/pom.xml

```diff
@@ -48,10 +48,22 @@
     <dependency>
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-core_${scala.binary.version}</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-client</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
     <dependency>
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-sql_${scala.binary.version}</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.orc</groupId>
+          <artifactId>orc-core</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
Will this break ORC support in Spark and Hudi?

Reply:
Im not sure if this is more of test issue or production issue. For example ive seen orc related tests fail for this dependency conflict

```
Java.lang.NoSuchMethodError: org.apache.orc.TypeDescription.createRowBatch(I)Lorg/apache/hadoop/hive/ql/exec/vector/VectorizedRowBatch;

```

My understanding is that it seems to do with the hive 2 orc https://github.com/apache/hive/blob/rel/release-2.3.1/pom.xml  and the spark 3 orc https://github.com/apache/spark/blob/v3.1.3/pom.xml#L139 being different versions that dont work well together. 

In the original spark 3.2 pr https://github.com/apache/hudi/pull/4752 the same orc issues were present and we made a call then to disable the orc related tests.

Reply:
cc @xushiyan are you familiar with the specifics of this conflict?

Reply:
Got it, could you manually verify if ORC format still works with Spark bundle?